client/sections.js

```diff
@@ -735,6 +735,7 @@ const sections = [
 			'/marketplace/hosting',
 			'/marketplace/checkout',
 			'/marketplace/assign-license',
+			'marketplace/download-products',
```

Comment:
[non-blocker] It has a missing slash in the beginning.

```suggestion
			'/marketplace/download-products',
```